Prototype/Components/newfooter/newfooter.html

```diff
@@ -0,0 +1,91 @@
+<div class="cmp-new-footer">
+    <div class="cmp-new-footer__main-container">
+        <div class="cmp-new-footer__top-section">
+            <div class="cmp-new-footer__top-section-items">
+                <h2 class="cmp-new-footer__title">HOMEBUYERS</h2>
```

Comment:
If design calls for caps case, I'd prefer leaving text title case (Homebuyers) and use CSS `text-transform: uppercase;`